### PR TITLE
(GER) ASOP terminal + Minor elevator + part of hull scraping

### PR DESCRIPTION
**Pull request by Auhrus**
ASOP terminals translated
Minor improvements on the elevator ---- Replaced the word "Stockwerk" to "Etage"
accidentally translated a small part for hull scraping 😄

### CODE REVIEW

File: data/Localization/german_(germany)/global.ini [hunk 1]
```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
 ShipName_Test_001=Freiberufler
 ShipName_Test_002=Starfarer
 ShipName_Test_003=Konstellation
-ShipSelector_ButtonTitle_Deliver=Lieferung anfordern
-ShipSelector_ButtonTitle_FileClaim=Einen Anspruch anmelden
-ShipSelector_ButtonTitle_PayDeductible=Selbstbehalt zahlen
-ShipSelector_ButtonTitle_PayExpedited=Expedited-Gebühr bezahlen
+ShipSelector_ButtonTitle_Deliver=Bestellen
```
Comment (ROBdk97) [inline data/Localization/german_(germany)/global.ini:39710]: How about ShipSelector_ButtonTitle_Deliver=Anfordern

File: data/Localization/german_(germany)/global.ini [hunk 2]
```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
...
+ShipSelector_ClaimLabel_Claim=Beanspruchen
+ShipSelector_ClaimLabel_Deductible=Selbstbeteiligung
+ShipSelector_ClaimLabel_Deliver=Bestellen
 ShipSelector_ClaimLabel_Delivering=Unterwegs
-ShipSelector_ClaimLabel_Expedite=Expedit
-ShipSelector_ClaimLabel_Expedited=Versandt
-ShipSelector_Crew=Besatzung
-ShipSelector_DeliverDescription=Nach der Übermittlung wird das folgende Fahrzeug innerhalb des geschätzten Zeitrahmens an Ihren aktuellen Standort geliefert.
+ShipSelector_ClaimLabel_Expedite=Beschleunigen
+ShipSelector_ClaimLabel_Expedited=Versendet
```
Comment (ROBdk97) [inline data/Localization/german_(germany)/global.ini:39720]: ShipSelector_ClaimLabel_Expedited=Beschleunigt

File: data/Localization/german_(germany)/global.ini [hunk 3]
```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
...
 ShipSelector_DeliveryDescription=Dieses Fahrzeug wird gerade ausgeliefert und ist auf dem Weg zu Ihnen.
-ShipSelector_DeliveryTime=Lieferfrist
-ShipSelector_ExpeditedDescription=Die Bearbeitung des Antrags wurde beschleunigt.
-ShipSelector_ExpeditedFee_Label=Expedited-Gebühr:�
-ShipSelector_ExpeditedTime_Instant=Expedited Zeit: Sofort
-ShipSelector_ExpeditedTime_Label=Eilige Zeit:�
-ShipSelector_FileClaimDescription=Nach der Übermittlung wird das folgende Fahrzeug in der geschätzten Zeit an Ihren aktuellen Standort geliefert.
+ShipSelector_DeliveryTime=Lieferzeit
+ShipSelector_ExpeditedDescription=Die Bearbeitung der Beanspruchung wurde beschleunigt.
+ShipSelector_ExpeditedFee_Label=Selbstbeteiligung-Gebühr:�
```
Comment (ROBdk97) [inline data/Localization/german_(germany)/global.ini:39727]: ShipSelector_ExpeditedFee_Label=Beschleunigungsgebühr

File: data/Localization/german_(germany)/global.ini [hunk 4]
```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
...
+ShipSelector_ExpeditedTime_Label=Lieferzeit Zeit:�
+ShipSelector_FileClaimDescription=Nach der Beanspruchung wird das folgende Fahrzeug in der geschätzten Zeit an Ihren aktuellen Standort geliefert.
 ShipSelector_Fine=Gut:
 ShipSelector_Header_Claims=Anspruch Fahrzeugversicherung
 ShipSelector_Header_Deliver=Anfrage zur Fahrzeugauslieferung
-ShipSelector_Header_Impounded=Beschlagnahmtes Fahrzeug reklamieren
-ShipSelector_Header_Main=Fuhrparkleiter
-ShipSelector_Header_RetrievalFailed=Fahrzeug nicht zurückerhalten
-ShipSelector_Header_Retrieved=Erfolgreich abgerufenes Fahrzeug
+ShipSelector_Header_Impounded=Beschlagnahmtes Fahrzeug ausbezahlen
```
Comment (ROBdk97) [inline data/Localization/german_(germany)/global.ini:39734]: ShipSelector_Header_Impounded=Fahrzeug zurückfordern
oder
ShipSelector_Header_Impounded=Beschlagnahmtes Fahrzeug beanspruchen

File: data/Localization/german_(germany)/global.ini [hunk 5]
```diff
@@ -39707,79 +39707,79 @@ ServiceBeacon_Revive_Obj_Title=Rescue ~serviceBeacon(InitiatorName)
...
+ShipSelector_PositionInQueue=Position in Warteschlange:
+ShipSelector_PoweredBy=BESCHLEUNIGTER SERVICE, POWERED BY:
+ShipSelector_RentalExpired=ABGELAUFEN
+ShipSelector_Retrieving_Description=Bitte warten Sie, während Ihr Fahrzeug ausgeparkt wird.
 ShipSelector_Status=Status
 ShipSelector_StatusNotApplicable=--
 ShipSelector_Vehicle=Fahrzeug
 ShipSelector_VehicleDelivered=Geliefert
-ShipSelector_WaitingForService=Bereithalten, um die Liste der verfügbaren Fahrzeuge aufzurufen.
+ShipSelector_WaitingForService=Liste iherer Schiffe wird aufzurufen...
```
Comment (ROBdk97) [inline data/Localization/german_(germany)/global.ini:39762]: ShipSelector_WaitingForService=Bitte warten, verfügbare Fahrzeuge werden abgerufen.
or
ShipSelector_WaitingForService=Liste Ihrer Schiffe wird abgerufen...
or
ShipSelector_WaitingForService=Liste Ihrer Schiffe wird aufgerufen...

Comment (ROBdk97) [reply data/Localization/german_(germany)/global.ini:39762]: "iherer" is still wrong xD

File: data/Localization/german_(germany)/global.ini (hunk 1 above)
Comment (Auhrus) [reply data/Localization/german_(germany)/global.ini:39710]: Yes, it fits and was adopted.

File: data/Localization/german_(germany)/global.ini (hunk 2 above)
Comment (Auhrus) [reply data/Localization/german_(germany)/global.ini:39720]: Was also adopted

File: data/Localization/german_(germany)/global.ini (hunk 3 above)
Comment (Auhrus) [reply data/Localization/german_(germany)/global.ini:39727]: Was also adopted

File: data/Localization/german_(germany)/global.ini (hunk 5 above)
Comment (Auhrus) [reply data/Localization/german_(germany)/global.ini:39762]: I have no idea what was going on, I somehow misread it myself.🤣

File: data/Localization/german_(germany)/global.ini (hunk 4 above)
Comment (Auhrus) [reply data/Localization/german_(germany)/global.ini:39734]: Was also changed.

**Review by ROBdk97 — APPROVED** [review]
No, not yet xD

**Comment by Dymerz** [comment]
@all-contributors Please add @Auhrus for translation

**Comment by Dymerz** [comment]
@all-contributors Please add @ROBdk97 for projectManagement

**Pull request merged**